xbmc/interfaces/json-rpc/PVROperations.cpp

```diff
@@ -251,6 +251,17 @@ JSONRPC_STATUS CPVROperations::Scan(const std::string &method, ITransportLayer *
   return ACK;
 }
 
+JSONRPC_STATUS CPVROperations::SearchMissingChannelIcons(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  if (!g_PVRManager.IsStarted())
+    return FailedToExecute;
+
+  else
```

Comment:
no need to add the `else` because of the earlier return if the manager is not started and it looks a little bit odd with this empty line before.